docs/_docs/templates.md

```diff
@@ -558,19 +539,29 @@ You can also use this tag to create a link in Markdown as follows:
 {% endraw %}
 ```
 
-### Post URL
+Including `{% raw %}{{site.baseurl}}{% endraw %}` is optional &mdash; it depends on whether you want the link to be absolute or root-relative.
```

Comment:
This statement is wrong.  
Inserting `{{ site.baseurl }}` doesn't make the URL absolute, its made so by: `{{ site.url }}/{{ site.baseurl }}`  
Jekyll 3.3 ships with a new set of filters to address this confusion. Simply use `{{ "file" | relative_url }}` or `{{ "file" | absolute_url }}`